projects/crypto-currency/destemidos/Api/routes/singin.js

```diff
@@ -1,7 +1,26 @@
 import app from "./configs/app.js"
+import { Mongoose, UserSchema } from './configs/db.js'
 
-app.get('/login', (req,res)=>{
-    res.send('Fazer o login')
+app.post('/singin', async (req, res) => {
+ 
+  const email = req.body.email
+  const password = req.body.password
+  
+  const Users = Mongoose.model('users', UserSchema, 'users')
+  
+  try {
+    const foundUser = await Users.findOne({ email: email, password: password }).exec()
+    
+      if(foundUser) {
+        res.status(200).send('Login realizado com sucesso')
+        
```

Comment:
Poderia remover essa linha vazia?

```suggestion
      if(foundUser) {
        res.status(200).send('Login realizado com sucesso')
```

Reply:
removido a linha